src/lib/toolbar/toolbar.ts

```diff
@@ -51,12 +53,19 @@ export class MatToolbarRow {}
   preserveWhitespaces: false,
 })
 export class MatToolbar extends _MatToolbarMixinBase implements CanColor, AfterViewInit {
+  private _document: Document;
 
   /** Reference to all toolbar row elements that have been projected. */
   @ContentChildren(MatToolbarRow) _toolbarRows: QueryList<MatToolbarRow>;
 
-  constructor(elementRef: ElementRef, private _platform: Platform) {
+  constructor(
+    elementRef: ElementRef,
+    private _platform: Platform,
+    @Inject(DOCUMENT) document?: any) {
```

Comment:
@crisbeto I know this PR has already been merged, but shouldn't this have `@deletion-target x.y.z` as in others `constructor` parameters?

Reply:
I don't think we had the tooling around `@deletion-target` when I first did this PR.

Reply:
Aha, ok. Do you pretend to submit a PR to add this?